test/AspNetCoreModuleTests/utility_tests.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+#include "CppUnitTest.h"
+
+using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+
+namespace AspNetCoreModuleTests
+{
+    TEST_CLASS(UTILITY_TESTS)
+    {
+    public:
+
+        TEST_METHOD(PassUnexpandedString_ExpandsResult)
+        {
+            HRESULT hr = S_OK;
+            PCWSTR unexpandedString = L"ANCM_TEST_ENV_VAR";
+            PCWSTR unexpandedStringValue = L"foobar";
+            STRU   struExpandedString;
+            SetEnvironmentVariable(L"ANCM_TEST_ENV_VAR", unexpandedStringValue);
+
+            hr = UTILITY::GetExpandedEnvironmentString(L"%ANCM_TEST_ENV_VAR%", &struExpandedString);
```

Comment:
it will be better to put the input as "%ANCM_TEST_ENV_VAR%/test" to make sure the resolved string is still valid.
How about if user give two parameters and both contains environment variables?